src/components/App/index.jsx

```diff
@@ -116,7 +122,7 @@ const AppWrapper = () => {
             element={<PageWrap><UserActivationPage /></PageWrap>}
           />
           <Route
-            path="/:enterpriseSlug"
+            path="/:enterpriseSlug/admin?"
```

Comment:
Updated to handle regression introduced where paths at `/:enterpriseSlug/admin` will no longer redirect and throw a 404

The `?` references the `/admin` path as optional to handle 404 on the routes `/:enterpriseSlug/admin` to redirect to `/:enterpriseSlug/admin/learners` as expected